meetingRooms/src/main/java/com/meetingRooms/controller/userimportserv.java

```diff
@@ -0,0 +1,373 @@
+package com.meetingRooms.controller;
+
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.security.SecureRandom;
+import java.util.List;
+import java.util.Random;
+import java.util.UUID;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+
+import org.apache.commons.fileupload.FileItem;
+import org.apache.commons.fileupload.disk.DiskFileItemFactory;
+import org.apache.commons.fileupload.servlet.ServletFileUpload;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+
+import com.meetingRooms.entity.ImportUser;
+import com.meetingRooms.service.ImportUserServiceInterface;
+import com.meetingRooms.utility.ImportUserServiceFactory;
+import com.meetingRooms.utility.xmlEmailException;
+import com.meetingRooms.utility.xmlPhoneException;
+
+
+
+
+
+
+/**
+ * Servlet implementation class addxml
+ */
+public class userimportserv extends HttpServlet {
+	private static final long serialVersionUID = 1L;
+   
+    public userimportserv() {
+        super();
+        
+    }
+
+	/**
+	 * @see HttpServlet#doPost(HttpServletRequest request, HttpServletResponse response)
+	 */
+    protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        
+    	if(ServletFileUpload.isMultipartContent(request))
+    	
+    		{
+            
+    		try 
+            	
+            	{
+    			
+    			//path to create  DatabaseOfUser  you can change as per your work directory
+    			Path path = Paths.get("C:\\Users\\kunal\\Documents\\training\\CodeFury\\meetingRooms\\src\\main\\webapp\\DatabaseOfUsers");
+    			Files.createDirectories(path);
+    			
+    			
+    			//object creaation using factory method for service clas
+    			ImportUserServiceInterface si=ImportUserServiceFactory.createobject("adminservice");
+    		    
+    			
+    			//path where imported xml file from frontend will save
+    			File DIR = new File("c:/xmlFiles");
```

Comment:
Can we create the directory in our project itself, instead of absolute 'C:/xmlFiles'. Will not work on linux machines.